netbox/dcim/tables/power.py

```diff
@@ -87,6 +87,10 @@ class PowerFeedTable(TenancyColumnsMixin, CableTerminationTable):
         linkify=True,
         verbose_name=_('Tenant')
     )
+    site = tables.TemplateColumn(
```

Comment:
We should be able to do this with a regular column by setting its accessor to `rack__site` and `linkify=True`.